samples/client/petstore/typescript-fetch/builds/allOf-readonly/models/Club.ts

```diff
@@ -28,10 +28,10 @@ import {
 export interface Club {
     /**
      * 
-     * @type {ClubOwner}
+     * @type {Owner}
      * @memberof Club
      */
-    owner?: ClubOwner;
```

Comment:
Hmm, I hadn't even noticed this bug - it seems like something has dropped the `readOnly` flag in an `allOf` context. That's obviously fixed now here, but may need addressing elsewhere too.